src/libs/ReportUtils.ts

```diff
@@ -2933,11 +2963,9 @@ function buildOptimisticChatReport(
 
 /**
  * Returns the necessary reportAction onyx data to indicate that the chat has been created optimistically
- * @param {String} emailCreatingAction
- * @param {String} [created] - Action created time
- * @returns {Object}
+ * @param  [created] - Action created time
```

Comment:
CMD + F with '@param  ' (two spaces), change to '@param '.

![image](https://github.com/Expensify/App/assets/46095609/fea62d19-cd9b-4653-98d3-05b9848f2342)